### PR TITLE
LYNX-154: Documentation for attributeForms query

### PR DESCRIPTION
**Pull request by loginesta**
## Purpose of this pull request

This pull request (PR) is to add the documentation for GraphQl attributesForm query.
This PR is related to this jira ticket: [LYNX-101](https://jira.corp.adobe.com/browse/LYNX-101)

## Affected pages

New page: graphql/schema/attributes/queries/attributes-form.md

### CODE REVIEW

File: src/pages/graphql/schema/attributes/queries/attributes-form.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+---
+title: attributesForm query | Commerce Web APIs
+---
+
+# attributesForm query
+
+The `attributesForm` query retrieves EAV attributes associated to a frontend `form`.
```
[the remark this answers is not in the document]
Comment (loginesta) [reply src/pages/graphql/schema/attributes/queries/attributes-form.md:7]: Regarding the possible values for this, you can use the values in the `form_code` column from `customer_form_attribute` table.
Just for you to know, the resolver that processes the graphql query retrieves form attributes associated to customer and does not allow to retrieve attibutes that belong to `admin` forms.

As far as I know, there is no query that returns the list of forms associated to customers, they are hardcoded and it doesn't look like the list will grow.

Currently, they are: `customer_account_create`, `customer_account_edit` and `customer_register_address`.
Not sure if it's worthy to add this information in the documentation page.

Let me know your thoughts on this. Thanks

**Comment by loginesta** [comment]
@keharper i will double check it with the team! thanks for your feedback

**Comment by sivaschenko** [comment]
@keharper @loginesta this query can potentially support any forms, however, only customer form are currently dynamic (editable in the admin panel) so only customer (and custom address) form are supported. The query name should be `attributesForm`

File: src/pages/graphql/schema/attributes/queries/attributes-form.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (loginesta) [reply src/pages/graphql/schema/attributes/queries/attributes-form.md:7]: nice catch @keharper

**Review by keharper — APPROVED** [review]

**Pull request merged**